keps/sig-scheduling/3521-pod-scheduling-readiness/README.md

```diff
@@ -979,6 +1025,8 @@ Think about adding additional work or introducing new steps in between
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
 
+This delay should be negligible.
```

Comment:
Please explain "this" - (I'm assuming you're talking about pod-startup-time which contains scheduling).

Also - I would say that this actually doesn't affect it, because the SLO explicitly talks about "schedulable" pods - if schedulingGate is set it's unclear if this pod is actually schedulable ...

Reply:
Thanks. I took a look at https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#kubernetes-slisslos, a SLO about the time from pod creation to pod scheduled is not defined officially yet, so the answer is No here.